src/containerapp/azext_containerapp/custom.py

```diff
@@ -3322,3 +3322,43 @@ def list_maintenance_config(cmd, resource_group_name, env_name):
     )
     r = maintenance_config_decorator.list()
     return r
+
+
+def update_httprouteconfig(cmd, resource_group_name, name, httprouteconfig_name, yaml):
```

Comment:
Please add help description for your command parameters according to https://github.com/Azure/azure-cli/blob/main/doc/authoring_command_modules/authoring_commands.md#customize-arguments